tmt/base.py

```diff
@@ -1125,6 +1140,15 @@ def _filters_conditions(self, nodes, filters, conditions):
             except fmf.utils.FilterError as error:
                 # Handle missing attributes as if filter failed
                 continue
+            # Links
+            try:
+                # Links are in OR relation
+                if links and all([not node.has_link(link_)
+                                 for link_ in links]):
```

Comment:
Why are you using `link_` instead of `link`?

Reply:
For me link_ is more clear as temporary variable and it is easier to distinguish from links

Reply:
As a temporary variable... interesting... ok.